core-blocks/code/theme.scss

```diff
@@ -0,0 +1,8 @@
+.wp-block-code {
+	font-family: $editor-html-font;
+	font-size: $text-editor-font-size;
+	color: $dark-gray-800;
+	padding: .8em 1.6em;
+	border: 1px solid $light-gray-500;
+	border-radius: 4px;
+}
```

Comment:
Next person to edit this file, if they're using the provided EditorConfig, will introduce a trailing new line to the file.

I'd suggest installing a plugin for your editor so we can be consistent on this:

https://editorconfig.org/#download

Reply:
Found it. Turns out that's a VSCode setting that defaults to off, which confused me because I switched from Atom where it defaults to on:

![screen shot 2018-07-06 at 15 27 41](https://user-images.githubusercontent.com/1204802/42381528-9f656bfc-8131-11e8-8ce4-852361c2423c.png)

I've now fixed this, and made a backup of my user config. Sorry about that.